app/index.html

```diff
@@ -14,6 +14,14 @@
                 </a>
             </div>
 
+            <div id="top-search">
+                <img id="top-search-icon" src="img/search.png" />
+            </div>
+            
+            <div id="top-search-box">
+                <input type="text" id="top-search-input" placeholder="Search Google..."></input>
+            </div>
```

Comment:
If I change it to:
`<div id="top-input">
                <input type="text" id="top-input-box" placeholder="Search Google..."></input>
            </div>`

Would that work?